lib/bulk/common.js

```diff
@@ -2,181 +2,220 @@
 
 const Long = require('mongodb-core').BSON.Long;
 const MongoError = require('mongodb-core').MongoError;
-const util = require('util');
 const toError = require('../utils').toError;
 const handleCallback = require('../utils').handleCallback;
+const applyWriteConcern = require('../utils').applyWriteConcern;
+const shallowClone = require('../utils').shallowClone;
+const ObjectID = require('mongodb-core').BSON.ObjectID;
 
 // Error codes
-var UNKNOWN_ERROR = 8;
-var INVALID_BSON_ERROR = 22;
-var WRITE_CONCERN_ERROR = 64;
-var MULTIPLE_ERROR = 65;
+const UNKNOWN_ERROR = 8;
+const INVALID_BSON_ERROR = 22;
+const WRITE_CONCERN_ERROR = 64;
+const MULTIPLE_ERROR = 65;
 
 // Insert types
-var INSERT = 1;
-var UPDATE = 2;
-var REMOVE = 3;
+const INSERT = 1;
+const UPDATE = 2;
+const REMOVE = 3;
 
 /**
  * Helper function to define properties
  * @ignore
  */
-var defineReadOnlyProperty = function(self, name, value) {
+function defineReadOnlyProperty(self, name, value) {
```

Comment:
You should be able to remove this completely if all the classes in this file are converted to using getters